app/controllers/miq_policy_controller.rb

```diff
@@ -13,6 +13,15 @@ class MiqPolicyController < ApplicationController
   after_action :cleanup_action
   after_action :set_session_data
 
+  # These all should typically be the same but the separation is useful when landing support in parts.
```

Comment:
@cben please implement the compliance bit and avoid the separation as well as CONDITION_FOLDERS.
(is there and problem with the compliance bits?)

:+1: for moving MiqPolicy::UI_FOLDERS to this contoller